### DISCUSSION TITLE
2주차 회고

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#39] [회고]
### KPT 회고는 세가지 관점으로 회고를 진행합니다.
- KEEP(현재 만족하고 있고, 계속 이어가길 바라는 점)
- PROBLEM(불편을 느껴 개선이 필요한 점)
- TRY(PROBLEM의 해결책으로 다음 회고까지 실행할 점)

아래 양식을 복사해 댓글로 작성해주시면 됩니다.
```
### KEEP

### PROBLEM

### TRY
```

**Comment by seolsis** [comment]
### KEEP
- 현재 진행 속도
- github를 최대한 활용하는 점
- 기획 단계에서 사용자를 최우선사항으로 고려한 점
### PROBLEM
- Flow chart를 개발 시작 전에 완성하지 못한 점
### TRY

**Comment by Seoin02** [comment]
### KEEP
- 꼼꼼하고, 힘이 되는 코드리뷰 문화
- 깃허브 이슈, PR, 액션, Discussions 활용
- 각자 맡은 역할을 열심히 하는 팀원들
- 모르는 것을 서로 알려주고, 도와주는 점
- 회의에서 의견을 적극적으로 내고, 피드백 수용이 빠르다.
- 코어타임을 잘 지키고, 소통이 잘 된다.
- 초기세팅을 잘해놔서 작업하기 원활하다.

### PROBLEM
- 노션에 데일리스크럼 작성이 잘 안됐다. 접근성, 가독성이 안 좋았던 게 원인이라고 생각한다.
- 공통 컴포넌트를 만들 땐 다양한 사용처를 고려해야 하기 때문에 PR이 올라오면 잘 확인하면 좋을 듯 하다.

### TRY
- 깃허브 Discussions를 활용해 데일리스크럼을 작성한다.
- 공통 컴포넌트 PR은 확인하려고 노력한다.

**Comment by q45402sk** [comment]
### KEEP
- 팀원 모두가 팀미팅 시간을 잘 지킨다. 
- 대부분 작업 속도가 빠른 것 같다. 
- 소통이 잘 되고, 서로 예의를 잘 지킨다. 

### PROBLEM
- PR을 잘 확인하지 못하는 팀원이 있다. (뜨끔)

### TRY
- PR이 올라오면 바로바로 확인하려고 노력한다.

**Comment by devwqc** [comment]
### KEEP
- 디자이너, 백엔드 분들을 포함한 모두 소통이 잘 된다.
- 기획 및 와이어 프레임이 빠르게 나와서 작업이 빠르게 진행되고 있다.
- 코어 타임을 잘 지키며 다른 일정이 있을 때 미리 알려준다.
- 서로 어려운 일을 떠넘기려 하지 않고 맡아서 진행한다.

### PROBLEM
- 바쁘다는 핑계로 워크 플로우 및 데일리 스크럼 작성이 늦어진 점이 죄송하다.

### TRY
- 소통에 중요한 데일리 스크럼 작성은 바로 진행해야겠다.

**Comment by minyoung0503** [comment]
### KEEP
* 소통이 빠르게 잘 된다.
* 자신과 관련없는 일이라도 같이 고민하고 해결하는데 도움을 준다.
* 팀 분위기가 밝고 적극적이다.
* 사소한 거 하나하나 다 기록해주셔서 나중에 까먹더라도 다시 확인할 수 있다.

### PROBLEM
* 코드리뷰를 많이 하지 못했다.

### TRY
* 코드리뷰를 더 적극적으로 해야겠다.
